bigbluebutton-html5/imports/api/audio/client/bridge/kurento.js

```diff
@@ -0,0 +1,65 @@
+import BaseAudioBridge from './base';
+
+const MEDIA = Meteor.settings.public.media;
+const MEDIA_TAG = MEDIA.mediaTag.replace(/#/g, '');
+
+export default class KurentoAudioBridge extends BaseAudioBridge {
+  constructor(userData) {
+    super();
+    const {
+      userId,
+      username,
+      voiceBridge,
+      internalMeetingID,
+    } = userData;
+
+    this.user = {
+      userId,
+      name: username,
+    };
+
+    this.internalMeetingID = voiceBridge;
```

Comment:
Perhaps you want `internalMeetingID` here, not `voiceBridge`?

Reply:
I need both, but yeah, it is convoluted. I'll hone it.

Reply:
Oh, I misread it. Fixed.